### DISCUSSION TITLE
Chinese document translation.

### DISCUSSION THREAD

**Discussion by Alfred-Skyblue** [vueuse/vueuse#2832] [Show and tell]
[Chinese document](https://alfred-skyblue.gitee.io/)

I like this project very much. In order to facilitate users to view the documents, I have translated the official documents into Chinese. I hope the author can link to the document address, thank you.

**Comment by antfu** [comment]
感谢翻译！

但是很抱歉我们无法推荐它，因为翻译的文档并没有遵照原文的形式（不同的 theme 与颜色，不同的 Demo 格式等）与用词（引入了 Hooks 这个不存在 VueUse 中的概念等）。这会导致中文文档出现与官方的体验断层。建议在文档中提及“社区维护的中文文档”。

**Reply by chaochaoer** [reply]
你好，我fork了代码到我的仓库，目前翻译了一大部分。在我决定翻译文档的时候，我并不知道社区已经有了对应的成果，因为当时我没有在百度上搜到有关“vueuse中文文档”的中文网站，所以我就自己注册了域名，并且在“关于本站”模块中 http://www.vueusejs.com/about-me.html 备注了并非官方文档字样，网站地址是 https://www.vueusejs.com

**Reply by Alfred-Skyblue** [reply]
你的翻译是更好的
